### PR TITLE
Move spark config to a visible cell

### PR DESCRIPTION
**Pull request by tamargrey**
- Seeing the docs build fail on the `copy-schema` PR, so hoping to recreate that here and find a fix.
- The fix involves making the spark config visible, and this isn't ideal, but it seems like it's not getting executed in a raw rst cell in the same way it does in a regular cell.

### CODE REVIEW

File: docs/source/release_notes.rst [hunk 1]
```diff
@@ -26,6 +26,7 @@ Release Notes
         * Store column typing information in a ``ColumnSchema`` object instead of a dictionary (:pr:`791`)
     * Documentation Changes
         * Update Pygments version requirement (:pr:`751`)
+        * Move spark config to a visible cell (:pr:`801`)
```
Comment (thehomebrewnerd) [inline docs/source/release_notes.rst:29]: What if we updated this line to say `Update spark config for docs build` and mention this pr and 787 from below, and then update the testing changes entry to just say `Update spark config for test fixtures` to separate these out better?

**Review by thehomebrewnerd — COMMENTED** [review]
Do you think we should add a comment somewhere to let readers know that this spark config cell is likely not something they will need to execute? I'm not sure how to best convey that, but I think it would be nice to somehow let them know this isn't something that is required for them to use if they already have spark configured properly.

**Comment by tamargrey** [comment]
> Do you think we should add a comment somewhere to let readers know that this spark config cell is likely not something they will need to execute? I'm not sure how to best convey that, but I think it would be nice to somehow let them know this isn't something that is required for them to use if they already have spark configured properly.

Maybe a comment above indicating that and also provide a link to some sort of resource that explains what it is so that if they did happen to need to run it they can figure that out for themselves.

File: docs/source/release_notes.rst (hunk 1 above)
Comment (tamargrey) [reply docs/source/release_notes.rst]: that makes sense to me!

**Review by thehomebrewnerd — APPROVED** [review]
LGTM!

**Pull request merged**